packages/lib/passport-interface/test/SubscriptionManager.spec.ts

```diff
@@ -33,6 +34,67 @@ describe("Subscription Manager", async function () {
     MockDate.reset();
   });
 
+  function setupTestManager(
+    subs: { provider: string; feed: string }[]
+  ): FeedSubscriptionManager {
```

Comment:
one unspecified precondition of this function is that it doesn't support passing in duplicate entries via `subs`

Reply:
True.  It's a test helper, so I don't think it needs thorough docs.  If someone used it in a way which broke that assumption the expect() below would tell them what to fix.